client/components/credit-card/stored-card.scss

```diff
@@ -0,0 +1,46 @@
+// Credit Card Payment Box
+// -----------------------------------
+
+.credit-card__stored-card {
+	background-image: url('/calypso/images/upgrades/cc-visa-disabled.svg');
+	background-position: 18px center;
+	background-repeat: no-repeat;
+	background-size: 60px auto;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	padding: 15px 15px 15px 95px;
+
+	.credit-card__stored-card-name {
```

Comment:
Did we need to group this under the parent for the added CSS specificity? It's okay to have this as top level otherwise.

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/24485/commits/90d6d02d50eaba69bbb03bf175d04e20cf5f47cd